src/run/next.cts

```diff
@@ -96,13 +117,17 @@ export async function getMockedRequestHandlers(...args: Parameters<typeof getReq
       } catch (error) {
         // only try to get .html files from the blob store
         if (typeof path === 'string' && path.endsWith('.html')) {
+          const fsBlobsManifest = await getFSBlobsManifest()
+
           const store = getRegionalBlobStore()
-          const relPath = relative(resolve('.next/server/pages'), path)
+          const relPath = relative(resolve(join(fsBlobsManifest.outputRoot, '/server/pages')), path)
```

Comment:
I think this was not actually working correctly if user had non-default `distDir` (aka not `.next`) and also likely not working in monorepos

Reply:
this makes sense, however the `outputRoot` seems to have nothing to do with blob manifest so I'm wondering if there should be a different way to specify this value?

Reply:
given https://github.com/netlify/next-runtime/pull/2649#issuecomment-2416419879 - I'll drop this change and keep hardcoded `.next` here as out of scope for this PR - I still think this is an issue, but I will not try to fix it in same PR that introduces creation of blob cache entries for fallbacks